### PR TITLE
More Branch ids and device fingerprint fix

### PR DESCRIPTION
**Pull request by aaustin**
@qinweigong

### CODE REVIEW

File: Branch-SDK/Branch-SDK/BNCSystemObserver.m [hunk 1]
```diff
@@ -42,6 +42,18 @@ + (NSString *)getUniqueHardwareId:(BOOL *)isReal {
...
++ (BOOL)adTrackingSafe {
+    Class ASIdentifierManagerClass = NSClassFromString(@"ASIdentifierManager");
+    if (ASIdentifierManagerClass) {
+        SEL sharedManagerSelector = NSSelectorFromString(@"sharedManager");
+        id sharedManager = ((id (*)(id, SEL))[ASIdentifierManagerClass methodForSelector:sharedManagerSelector])(ASIdentifierManagerClass, sharedManagerSelector);
+        SEL advertisingEnabledSelector = NSSelectorFromString(@"isAdvertisingTrackingEnabled");
+        BOOL enabled = ((BOOL (*)(id, SEL))[sharedManager methodForSelector:advertisingEnabledSelector])(sharedManager, advertisingEnabledSelector);
+        return enabled;
+    }
+    return YES;
```
Comment (qinweigong) [inline Branch-SDK/Branch-SDK/BNCSystemObserver.m]: By default, we return YES?

File: Branch-TestBed/Branch-TestBed/AppDelegate.m [hunk 2]
```diff
@@ -12,7 +12,8 @@ @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
-    Branch *branch = [Branch getInstance:@"5668720416392049"];
+    Branch *branch = [Branch getInstance:@"5680621892404085"];
```
Comment (qinweigong) [inline Branch-TestBed/Branch-TestBed/AppDelegate.m]: Change back to prod id?

Comment (aaustin) [inline Branch-TestBed/Branch-TestBed/AppDelegate.m]: oops

File: Branch-SDK/Branch-SDK/BNCSystemObserver.m (hunk 1 above)
Comment (aaustin) [inline Branch-SDK/Branch-SDK/BNCSystemObserver.m]: Right now, the default is YES. We only set NO if we can explicitly know that the user has opted out of ads

**Comment by qinweigong** [comment]
:+1:

**Pull request merged**